pom.xml

```diff
@@ -68,16 +68,16 @@
 
   <properties>
     <!--Dependency versions-->
-    <arpnetworking.commons.version>1.9.0</arpnetworking.commons.version>
```

Comment:
Let's get the new commons without the maven-javassist-core dependency in here first.

Reply:
We can do that before releasing metrics-client-java, I don't see why it should block this MR which has several other changes.